### DISCUSSION TITLE
请问如何才能统计微信某个小程序的时间？但是不统计微信的时间？

### DISCUSSION THREAD

**Discussion by xuexicn** [Planshit/Tai#161] [Q&A]
我想统计某个小程序的使用时间，但是不统计整个微信的时间，
小程序的名称是固定的，但是pid不固定。

请问，这个要怎么才能实现呢？
![wechat](https://user-images.githubusercontent.com/50934015/231331413-9fb267c5-9ddf-4b3c-b40b-a68cf341f2cf.png)
![wechat2](https://user-images.githubusercontent.com/50934015/231331414-867d3d6e-9811-4d2b-83fd-092c28192f59.png)

**Comment by noberumotto** [comment]
暂时不支持统计小程序的时间，未来看需求量大的话可能会以插件形式提供
